samples/apps/copilot-chat-app/webapi/Storage/ChatMessageRepository.cs

```diff
@@ -1,6 +1,6 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
-using SemanticKernel.Service.Skills;
+using SemanticKernel.Service.Model;
```

Comment:
```suggestion
using SemanticKernel.Service.Storage;
```

Reply:
Are you suggesting moving the classes to under Storage/?

Reply:
It looks like they are under Storage. ? 
![image](https://user-images.githubusercontent.com/56417140/234725773-415533ad-673f-46e3-894c-65b378f063eb.png)